packages/notifier/test/publish-kit.test.js

```diff
@@ -82,6 +82,8 @@ const assertCells = (t, label, cells, publishCount, expected, options = {}) => {
     }
   } else {
     const { tail: _tail, ...props } = firstCell;
+    // We need an element and an index here, which for..of does not give us in one go
+    // eslint-disable-next-line github/array-foreach
     cells.slice(1).forEach((cell, i) => {
       t.like(cell, props, `${label} cell ${i + 1} must match cell 0`);
     });
```

Comment:
> Consider a regular for loop

@turadg Aesthetically, I like `arr.forEach((el, i) => …)` much better than `let i = -1; for (const el of arr) { i += 1; … }` or [_even worse when the array is computed, as it is in this case_] `const arr = $deriveArrayFromInput(…); for (let i = 0; i < arr.length; i++) { const el = arr[i]; … }`. Frankly, this is the kind of code that most argues against the [`github/array-foreach` rule](https://github.com/github/eslint-plugin-github/blob/main/docs/rules/array-foreach.md).

There is another possibility that works because we're dealing with a plain array (whose only non-index property is "length", which is not enumerable), but it's even more bizarre:
```suggestion
    for (const [i, cell] of Object.entries(cells.slice(1))) {
      t.like(cell, props, `${label} cell ${Number(i) + 1} must match cell 0`);
    }
```

All things considered, I prefer the lint rule suppression and continued use of `forEach`, but I'm interested in your elaboration if you disagree.

Reply:
I agree